woocommerce/woocommerce-android
# issue/6111-step3-site-picker-changes

Part 3 of #6111.

## Changes
- This PR just adds the site picker to the nav graph and updates the logic in `MainActivity` to open the site picker fragment. - This just adds the redirect logic to an empty screen. The actual UI + business logic for the site picker will be taken care of in another PR. 
- This PR is part of WIP and will be merged into a feature branch so it's fine to handle the logic in a separate PR.

### Testing instructions
- Click on the Switch Store option and verify that you are redirected to the Site Picker fragment.
- Login to the app and notice that you are redirected to the Site Picker fragment.
